subprojects/docs/src/docs/userguide/userguide_single.adoc

```diff
@@ -0,0 +1,213 @@
+// Copyright 2018 the original author or authors.
```

Comment:
I guess we don't need `userguide.xml` any more.

Reply:
Yep. Good call.

Reply:
✔️